manim/logger.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import configparser
+
+from rich.logging import RichHandler
+from rich.console import Console
+from rich.theme import Theme
+#from .config import config
+
+#print(config)
+def parseTheme(fp):
+    config = configparser.ConfigParser()
+    config.read(fp)
+    theme=dict(config['log.color'])
```

Comment:
Use the config-parser in `config.py` to do any parsing. 

Add the dictionary corresponding to `log.color` to `configparser.config` and/or `configparser.file_writer_config`  with the key `"log.color"` or something similar.

The `configparser.config` or `configparser.file_writer_config` dictionaries can be used for any config data that needs to be accessible by `logger`. 

Mentioning @leotrs for his thoughts.

Reply:
There was a problem while I was doing so. `config.py` imports this file while running and trying to import using your method creates an import error as the root module `manim` would has not been initiated and there would be an circular import.

Reply:
This will be changed once the rich theme writer is turned into a full-fledged manim cfg writer.